netstandard/ref/System.cs

```diff
@@ -3764,14 +3913,19 @@ public partial struct TimeSpan : System.IComparable, System.IComparable<System.T
         public static System.TimeSpan FromSeconds(double value) { throw null; }
         public static System.TimeSpan FromTicks(long value) { throw null; }
         public override int GetHashCode() { throw null; }
+        public System.TimeSpan Multiply(double factor) { throw null; }
         public System.TimeSpan Negate() { throw null; }
         public static System.TimeSpan operator +(System.TimeSpan t1, System.TimeSpan t2) { throw null; }
+        public static System.TimeSpan operator /(System.TimeSpan timeSpan, double divisor) { throw null; }
+        public static double operator /(System.TimeSpan t1, System.TimeSpan t2) { throw null; }
         public static bool operator ==(System.TimeSpan t1, System.TimeSpan t2) { throw null; }
         public static bool operator >(System.TimeSpan t1, System.TimeSpan t2) { throw null; }
         public static bool operator >=(System.TimeSpan t1, System.TimeSpan t2) { throw null; }
         public static bool operator !=(System.TimeSpan t1, System.TimeSpan t2) { throw null; }
         public static bool operator <(System.TimeSpan t1, System.TimeSpan t2) { throw null; }
         public static bool operator <=(System.TimeSpan t1, System.TimeSpan t2) { throw null; }
+        public static System.TimeSpan operator *(double factor, System.TimeSpan timeSpan) { throw null; }
```

Comment:
It's the same as Line 3928, but with swapped arguments
`public static System.TimeSpan operator *(System.TimeSpan timeSpan, double factor) { throw null; }`

I think Line 3928 is better
`public static System.TimeSpan operator *(System.TimeSpan timeSpan, double factor) { throw null; }`

because it have the same argument order of the opposite operator in Line 3919
`public static System.TimeSpan operator /(System.TimeSpan timeSpan, double divisor) { throw null; }`

Reply:
This kind of multiplication is commutative, so I don't see any reason to force users to use a specific order of operands. What would be the advantage of forbidding e.g. `timeSpan * 4`?

Division is very different: `double / TimeSpan` doesn't make sense. Or, more accurately, the result would be frequency. E.g. `1 / TimeSpan.FromSeconds(1)` would have to return 1 Hz. And .Net doesn't have a type for representing that.

Reply:
You are right 👍